serde-generate/src/rust.rs

```diff
@@ -190,3 +191,57 @@ fn output_container(
         }
     }
 }
+
+pub struct Installer {
+    install_dir: PathBuf,
+}
+
+impl Installer {
+    pub fn new(install_dir: PathBuf) -> Self {
+        Installer { install_dir }
+    }
+
+    fn runtimes_not_implemented() -> Result<()> {
+        Err(std::io::Error::new(
```

Comment:
It doesn't change too much but you can always bring into scope `use std::io` or `use std::fs` so that you can drop the `std` everywhere. that's mostly a personal preference though

Reply:
Yeah, well, I usually try to stick with `std::io::Error` or `Error` but not `io::Error`. Also FWIW I used to `std::` everywhere in C++.